dom/src/main/javacc/org/apache/james/mime4j/field/datetime/DateTimeParser.jj

```diff
@@ -230,6 +230,10 @@ String year() :
       String year = (String) t.image;
       if (year.length() == 2) 
       {
+        if (year.charAt(0) > '6')
+        {
+          return "19" + year;
+        }

```

Comment:
Yes it will.

But we got time no? And since then, we would not care anymore about emails from 1970 I guess ^^